examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -77,10 +79,11 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
     traccc::spacepoint_formation sf(mng_mr);
```

Comment:
I don't know how it could be compiled but please look above lines - there are the lines that you should have changed rather than creating same algorithm objects
```
    traccc::clusterization_algorithm ca(mng_mr);
    traccc::spacepoint_formation sf(mng_mr);
```

Reply:
I think the algorithm is still using managed memory resource given that cpu clusterization takes too long time